server.js

```diff
@@ -549,12 +549,13 @@ camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
       var scheme = match[1];
       var serverUrl = match[2];  // eg, `sonar.qatools.ru`.
       var buildType = match[3];  // eg, `ru.yandex.qatools.allure:allure-core:master`.
-      var metric = match[4];
+      var metricName = match[4];
       var format = match[5];
-      var realMetric = metric === 'tech_debt' ? 'sqale_debt_ratio' : metric
```

Comment:
<https://img.shields.io/sonar/http/sonar.qatools.ru/ru.yandex.qatools.allure:allure-core/tech_debt.svg> is already in use. Could we check for `tech_debt` and switch `metricName` to `'sqale_debt_ratio'`? `tech_debt` seems unused.